lib/parity/backup.rb

```diff
@@ -31,6 +31,15 @@ def restore_from_development
     end
 
     def restore_to_development
+      drop_development_database
+      pull_remote_database_to_development
+    end
+
+    def drop_development_database
+      Kernel.system("dropdb #{development_db}")
```

Comment:
Heroku's PG tools and Parity are both already dependent on `pg_restore` and other Postgres utilities being installed and in the `$PATH`. Making use of `dropdb` isn't introducing a new dependency on users.

It might be worth thinking about if there's a way to bundle these utilities in the future, but I don't think we've seen reports of Parity failing from PG's utilities not being in the user's `$PATH`.